src/Microsoft.DotNet.Build.Tasks/CreateAzureContainer.cs

```diff
@@ -0,0 +1,63 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using Microsoft.WindowsAzure.Storage;
+using Microsoft.WindowsAzure.Storage.Blob;
+
+namespace Microsoft.DotNet.Build.Tasks
+{
+    public sealed class CreateAzureContainer : Task
+    {
+        [Required]
```

Comment:
There need to be comments about what these properties are and what format they are in.